content/installation/java/configuration/JavaAgentSystemProperties.md

```diff
@@ -175,9 +175,9 @@ By default, diagnostic logging is enabled, but set to the INFO level. It uses a
 
 ## More Information
 
-* [An Application Is Not Appearing In The List](troubleshooting-setup.html#missing)
+* [An Application Is Not Appearing In The List](troubleshooting-setup.html#missing) <!-- replace -->
 
-* [Getting Java Agent Logs](troubleshooting-java.html#java-logs)
+* [Getting Java Agent Logs](troubleshooting-java.html#java-logs) <!-- replace -->
 
```

Comment:
@marklacasse Two here

Reply:
For missing application not appearing in list.  https://support.contrastsecurity.com/hc/en-us/articles/360000464386

For Java logging: https://support.contrastsecurity.com/hc/en-us/articles/360035148152-How-to-get-logs-from-the-Java-agent